sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3518,6 +3518,7 @@ _ZNK2cl4sycl7sampler19get_addressing_modeEv
 _ZNK2cl4sycl7sampler33get_coordinate_normalization_modeEv
 _ZNK2cl4sycl7samplereqERKS1_
 _ZNK2cl4sycl7samplerneERKS1_
+_ZNK2cl4sycl8platform11get_backendEv
```

Comment:
@alexbatashev I am not fully understand this document: https://github.com/intel/llvm/blob/sycl/sycl/doc/ABIPolicyGuide.md

This PR needs to update the minor-number of the library major-minor version  as a non-breaking change, right?

Reply:
No, versions other than `V` are updated once per release. I'll prepare a pull-request with some clarifications to that document.